src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -1595,6 +1595,13 @@ internal static AmsiNativeMethods.AMSI_RESULT WinScanContent(string content, str
                         }
                     }
 
+                    if (warmUp)
```

Comment:
Why is this better than Init()?

Reply:
As explained above, the warm-up will create the `s_amsiSession` as well. And it prevents a race condition, even though the chance to hit that race condition is low.